test/workflows/components/workflows.libsonnet

```diff
@@ -249,6 +249,10 @@
                     name: "build-batcher",
                     template: "build-batcher",
                   },
+                  {
+                    name: "build-agent",
+                    template: "build-agent",
```

Comment:
I think it is missing the build template for `build-agent`, that's why the argo workflow is not triggered

Reply:
Thanks, will fix!